src/clib/lib/include/ert/enkf/enkf_defaults.hpp

```diff
@@ -96,7 +96,7 @@
 #define DEFAULT_ITER_RETRY_COUNT 4
 
 /* Default directories. */
-#define DEFAULT_RUNPATH "simulations/realization%d"
+#define DEFAULT_RUNPATH "simulations/realization<IENS>/iteration<ITER>"
```

Comment:
The FMU standard would be `"simulations/realization-<IENS>/iter-<ITER>"`. I suggest we change to that to avoid confusion when we have the opportunity.